### PR TITLE
Add joshuawilson to members list

### PR DESCRIPTION
**Pull request by joshuawilson**

### CODE REVIEW

File: peribolos/knative.yaml [hunk 1]
```diff
@@ -251,6 +251,7 @@ orgs:
     - joshlong
     - joshrider
     - joshua-bone
+    - joshuawilson
```
Comment (mattmoor) [inline peribolos/knative.yaml:254]: Want to join knative-sandbox while you are at it?

**Review by mattmoor — COMMENTED** [review]
/lgtm
/approve
/hold

Hold in case you want sandbox, otherwise feel free to `/unhold`

File: peribolos/knative.yaml (hunk 1 above)
Comment (joshuawilson) [reply peribolos/knative.yaml:254]: yes, please

**Review by mattmoor — COMMENTED** [review]
/lgtm
/approve

**Comment by joshuawilson** [comment]
@mattmoor can you remove the 'hold' or is there something else that needs to be done?

**Pull request merged**